ntbs-ui-tests/Helpers/Utilities.cs

```diff
@@ -11,7 +11,11 @@ public static class Utilities
     {
         public const int DRAFT_ID = 1;
         public const int NOTIFIED_ID = 2;
-
+        
+        public const string TBSERVICE_ABINGDON_COMMUNITY_HOSPITAL_ID = "TBS0001";

```

Comment:
do we want to get it from environment variable? or is it supposed to be like this here

Reply:
I'd have a preference to keeping test variables in the utilities files fwiw

Reply:
We also don't want a change in env variable to kill a test